docs/reference/commandline/exec.md

```diff
@@ -68,6 +68,17 @@ $ docker exec -it ubuntu_bash bash
 
 This will create a new Bash session in the container `ubuntu_bash`.
 
+Next, set an environment variable on the container.
```

Comment:
I know you clarify below that this variable only exists in the current bash session, but if you rephrased this to
"Next, set an environment variable in the current bash session"

It would help make things super clear.